integration_tests/src/main/python/hive_write_test.py

```diff
@@ -85,8 +85,6 @@ def do_write(spark, table_name):
 @pytest.mark.skipif(not is_hive_available(), reason="Hive is missing")
 @pytest.mark.parametrize("gens", [_basic_gens], ids=idfn)
 @pytest.mark.parametrize("storage_with_confs", [
-    ("PARQUET", {"spark.sql.legacy.parquet.datetimeRebaseModeInWrite": "LEGACY",
-                 "spark.sql.legacy.parquet.int96RebaseModeInWrite": "LEGACY"}),
```

Comment:
Why drop these?

Reply:
This is a fallback test. We now have full support for `LEGACY` in write thus we don't fallback anymore.

Reply:
This is similar: https://github.com/NVIDIA/spark-rapids/pull/9658/files#diff-33b5f03f5e0d5aa2b19d7666f2db31cb30b3c8d9b67271017ce960cc6ff40c78L789